.github/workflows/maze-runner.yml

```diff
@@ -0,0 +1,36 @@
+name: Maze Runner tests
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
I get github notifications when it fails

I'm not sure why it's specifically me that gets them though, I assume because I'm the last person to touch the repo?

>  Is it that it will run on the latest Symfony 5.x available each time, so could well change over time?

Yeah exactly, so we know if it starts failing on new versions. Otherwise we only know when we come to do other work so that gets sidetracked fixing the tests